### ISSUE TITLE
The avatar of a contact person doesn't load - Missing Image in e-mail details, contacts Mautic 5.1

### ISSUE THREAD

**Issue by stefan-franz** [mautic/mautic#13833]
### Mautic Version

5.0.x series

### Way of installing

I downloaded a release from https://www.mautic.org/mautic-releases

### PHP version

8.1.29

### What browsers are you seeing the problem on?

Firefox

### What happened?

In a mail, details, contacts, i have the contacts, but no image. Multi user instance. 
![Bildschirmfoto vom 2024-06-14 09-03-18](https://github.com/mautic/mautic/assets/96744684/fbbdb6d4-c5d7-47ef-8996-734e750f7ae1)

### How can we reproduce this issue?

Sending a mail

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by dsp76** [comment]
Could you please add details to the title: Its the avatar of a contact person, that doesn't load. Thanks!
Please also edit the version, its not working on 5.1.

Thanks!

**Comment by mautibot** [comment]
This issue has been mentioned on **Mautic Forums**. There might be relevant details there:

https://forum.mautic.org/t/mautic-5-1-avatar-broken/32378/2

**Comment by abhisekmazumdar** [comment]
I feel this is more a web server configuration issue, as mentioned in the Mautic forum: https://forum.mautic.org/t/mautic-5-1-avatar-broken/32378/6.

I tested this on both a ddev based local setup and Gitpod, and in both cases, I was able to see the profile images.

The steps I followed:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Go to the account (s/account).
3. Add a email ID which has a gravatar setup. Example: jitewaboh@lagify.com(from: https://stackoverflow.com/a/54004588)
4. And you should see the profile image.

**Comment by christopheg** [comment]
This is not a server issue.
The link in the page refers to this url:
https://www.gravatar.com/avatar/aa078506c365ca8c70ddd0fd01e8494b?s=250&d=https%3A%2F%2Fmautic.MYHOSTNAME.be%2Fapp%2Fassets%2Fimages%2Favatar.svg%3Fv5fca5e5d

This url responds with a 302 redirect to 
https://i2.wp.com/mautic.MYHOSTNAME.be/app/assets/images/avatar.svg?ssl=1

If this url is requested, I see the following request in my access log:
x.x.x.x - - [03/Jul/2024:13:41:16 +0200] "GET /app/assets/images/avatar.svg HTTP/1.0" 200 1269 "-" "Photon/1.0"

This requests returns a 200 with a proper image.
What is the purpose of sending the url of a default image to gravatar?

**Comment by pm-pmaas** [comment]
Hi can confirm this issue in 5.1.0 versions. I have the issue in composer based installs but also on non composer based installs. Before upgrade everything works fine but afterwards avator urls are broken ( in right top, as well in contact details ). The issue i have in more then 10 installations i upgraded till now.

![Scherm­afbeelding 2024-07-04 om 11 07 49](https://github.com/mautic/mautic/assets/17729447/c93362b4-1300-49aa-8b00-bc6fa88c1875)


I already tried the historic solution to change url path with an extra / or adding index.php in the site url settings, but this does not make sense.

![Scherm­afbeelding 2024-07-04 om 11 10 10](https://github.com/mautic/mautic/assets/17729447/52ce58db-e4dc-4b0c-a849-3e33ccfc7416)

**Comment by onlineexpert** [comment]
Hi can confirm this issue in 5.1.0 version after migration from Mautic 4 to 5.
I noticed that in 4 avatar.png was used and in 5 it is avatar.svg.
There seem to be many Gravatar related statements:
"SVG graphics are not allowed on WordPress.com for security reasons."
"SVG images are not supported as avatars"

So it's maybe just a matter of reverting back to the use of avatar.png in Mautic 5 or somehow deactivating the use of Gravatar?

**Comment by abhisekmazumdar** [comment]
I see these things after I investigated a little bit more:

1. No, the PNG or SVG error still gives 400 even though I changed the code to serve a PNG image here: https://github.com/mautic/mautic/blob/5.x/app/bundles/LeadBundle/Twig/Helper/DefaultAvatarHelper.php#L16
2. The purpose of sending the default image to Gravatar is that when an email is not found in Gravatar, it returns the provided image path. The image path which Gravatar generates gets redirected to an image path starting with https://i1.wp.com... which is where I see the issue happening(related to Jetpack setup in Gravatar), and it throws a 400 error.

We can add a fallback by checking if Gravatar returns a valid image. If not, then use the default one provided by `getDefaultAvatar()`.

**Comment by J-Wick4** [comment]
I vote to remove Gravatar completely from the code base and use a self-hosted option instead.

**Comment by AlbertMN** [comment]
We experience this issue too

**Comment by abhisekmazumdar** [comment]
The fix has been merged into [5.1](https://github.com/mautic/mautic/tree/5.1) branch.

**Comment by Ahson-Shaikh** [comment]
I'm using Docker Image 5.1-apache and the fix is still pending there.
